source/jormungandr/jormungandr/interfaces/v1/serializer/fields.py

```diff
@@ -218,6 +219,7 @@ class NoteSerializer(serpy.Serializer):
     id = jsonschema.Field(schema_type=str, display_none=True)
     value = jsonschema.Field(schema_type=str)
     category = jsonschema.Field(schema_type=str, schema_metadata={'enum': ['comment', 'terminus']})
+    comment_type = jsonschema.Field(schema_type=str, display_none=False)
```

Comment:
Same above: it could be call a `sub_category` so that we can use the field for other categories too.